CommonFindPackage.cmake

```diff
@@ -269,4 +269,8 @@ macro(common_find_package_post)
   endif()
   common_graph(${PROJECT_NAME})
   message(STATUS ${__configure_msg})
+
+  # If this variable was given in the command line, ensure that the package
+  # installation is only run in this cmake invocation.
+  unset(INSTALL_PACKAGES CACHE)
```

Comment:
Is there no way to move this to SubProject.cmake ? Some users of the Subproject feature may not use CommonFindPackage (or not correctly)